### PR TITLE
Chore: Update webpack dependencies

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

Updates webpack dependencies:

- `webpack`: `5.74.0` -> `5.75.0`
- `webpack-cli`:  `webpack-cli` -> `5.0.1`
- `webpack-dev-server`: `4.9.3` -> `4.11.1`

### Why is it needed?

Keep dependencies up to date.

### CODE REVIEW

File: packages/core/helper-plugin/package.json [hunk 1]
```diff
@@ -90,6 +90,6 @@
     }
   },
   "peerDependencies": {
-    "webpack": "^5.74.0"
+    "webpack": "^5.75.0"
```
Comment (joshuaellis) [inline packages/core/helper-plugin/package.json:93]: I'm not sure webpack is a peer dep of helper-plugin because it's shipped pre-bundled?

**Review by joshuaellis — CHANGES_REQUESTED** [review]
small Q

File: packages/core/helper-plugin/package.json (hunk 1 above)
Comment (gu-stav) [reply packages/core/helper-plugin/package.json:93]: Oh it is - this PR just updates the version. If we want to change that we can do it in a follow-up PR. I think your bundling PR already takes care of that?

**Review by joshuaellis — DISMISSED** [review]

**Review by joshuaellis — APPROVED** [review]

**Comment by gu-stav** [comment]
@joshuaellis Sorry had to rebase one last time 🤦🏼

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**